tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -232,3 +235,21 @@ def test_nodereplacement_proactive(self):
         assert ceph_cluster_obj.wait_for_rebalance(
             timeout=1800
         ), "Data re-balance failed to complete"
+
+
+@tier4b
+@skipif_no_lso
+@ignore_leftovers
+@bugzilla("1840539")
+class TestNodeReplacementLSO(ManageTest):
+    """
+    Node replacement twice on LSO:
+    node_x -> node_y
+    node_z -> node_x
+    """
+
+    def test_nodereplacenet_lso(self):
```

Comment:
can this be tested also on non LSO?
Even if not explicitly mentioned in the BZ, I think we should have the coverage

Reply:
Done